docs/qiita-16S-analysis.rst

```diff
@@ -3,218 +3,424 @@
+
+To create a Bray-Curtis dissimilarity principal coordinate plot, select select "pcoa" from the drop-down menu. The parameters will appear below the workflow diagram:
+
+.. figure::  images/bray_curtis_pcoa.png
+   :align:   center
+
+All of the parameter have automatically selected for you just click "Add Command". Once the command is added the workflow should appear as follows:
+
+.. figure::  images/bray_curtis_workflow.png
+   :align:   center
+
+Click the run button to start the process of the beta diversity analysis. The view will return to the original screen, while the beta diversity analysis job runs.
+
+Another tool for measuring beta diversity is unweighted beta diversity analysis. Unweighted beta diversity analysis is when the types but not quantity of each taxa is taken into consideration when comparing samples to one another. This differs from weighted analysis which takes into consideration both the amount and variety of taxa in a sample. 
+
+To perform an unweighted beta diversity analysis, select the rarefied "dflt_name - BIOM" artifact in the processing network and select "Process". Then select the "dflt_name - BIOM" artifact and select "beta_diversity" from the drop-down menu. The parameters will appear below the workflow diagram:
+
+.. figure::  images/unweighted_beta_diversity.png
+   :align:   center
+   
+Several parameters have been automatically selected for you. In the "p-metric" field enter "unweighted Unifrac" and in the "i-tree" field enter "/databases/gg/13_8/trees/97_otus.tree", and click "Add Command". 
+
+To create an unweighted beta analysis principal coordinate plot, select select "pcoa" from the drop-down menu. The parameters will appear below the workflow diagram:
+
+.. figure::  images/unweighted_pcoa.png
+   :align:   center
+
+All of the parameters have been automatically selected for you just click "Add Command". Once the command is added the workflow should appear as follows:
+
+.. figure::  images/unweighted_workflow.png
+   :align:   center
+
+Click the run button to start the process of the beta diversity analysis. The view will return to the original screen, while the beta diversity analysis job runs.
+
+A final way to analyze the beta diversity is through filtered unweighted beta diversity analysis. This allows you to filter out samples. 
+
+To filter the data, select the rarefied "dflt_name - BIOM" artifact in the processing network and select "Process". Then select the "dflt_name - BIOM" artifact and select "filter_samples" from the drop-down menu. The parameters will appear below the workflow diagram: 
+
+.. figure::  images/filtered_unweighted_filtering2.png
+   :align:   center
 
+Several parameters have been automatically selected for you. In the "p-where" field we are filtering out certain samples. In this case we wanted to filter our samples in which "'Subject= "Volunteer 3'", and click "Add Command". 
```

Comment:
Agree on the typo correction, but perhaps this is a good opportunity to add a phrase noting that all fields are case-sensitive and a link to a SQL lite query construction helper doc for now: http://www.sqlitetutorial.net/sqlite-where/

Once we have the GUI on top of this, before publication, then this part can be rewritten.